app/views/schools/participants/index.html.erb

```diff
@@ -5,107 +5,55 @@
-                                  classes: ["govuk-\!-padding-bottom-static-1 govuk-\!-margin-bottom-static-0"]) %>
-              </li>
-            <% end %>
-          </ul>
-          <br>
-        <% end %>
-      <% end %>
-
-      <% @participants.mentors.keys.sort_by(&:full_name).each do |mentor| %>
-        <div class="group-card" aria-label="Mentor group">
-          <h2 class="govuk-visually-hidden">Mentor group</h2>
-
-          <h3 class="govuk-heading-s">
-            Mentor <span class="govuk-visually-hidden">- <%= mentor.full_name %></span>
-          </h3>
-          <%= govuk_summary_list(actions: false, borders: nil) do |list| %>
-            <% list.with_row do |row| %>
-              <% row.with_key(text: govuk_link_to(mentor.full_name,
-                                             school_participant_path(id: mentor.participant_profile_id, school_id: @school),
-                                             no_visited_state: true),
-                         classes: ["govuk-!-font-weight-regular"]) %>
-              <% row.with_value(
-                   text: render(StatusTags::SchoolParticipantStatusTag.new(
-                     participant_profile: mentor.participant_profile,
-                     induction_record: mentor.induction_record,
-                     display_description: false))) %>
-            <% end %>
-          <% end %>
-
-          <h3 class="govuk-heading-s">
-            ECTs <span class="govuk-visually-hidden">mentored by <%= mentor.full_name %></span>
-          </h3>
+  </div>
+</div>
 
-          <%= govuk_summary_list(actions: false, borders: nil) do |list| %>
-            <% @participants.ects_mentored_by(mentor).sort_by(&:full_name).each do |ect| %>
-              <% list.with_row do |row| %>
-                <% row.with_key(text: govuk_link_to("#{ect.full_name}",
-                                               school_participant_path(id: ect.participant_profile_id,
-                                                                        school_id: @school),
-                                               no_visited_state: true),
-                           classes: ["govuk-!-font-weight-regular", "govuk-\!-padding-bottom-static-0"]) %>
-                <% row.with_value(
-                     text: render(StatusTags::SchoolParticipantStatusTag.new(
-                       participant_profile: ect.participant_profile,
-                       induction_record: ect.induction_record,
-                       display_description: false)),
-                     classes: ["govuk-\!-padding-bottom-static-0"]) %>
-              <% end %>
-            <% end %>
-          <% end %>
-        </div>
-        <br>
+<div class="govuk-grid-row wrapper-dashboard-v2">
+  <div class="govuk-grid-column-one-third">
+    <div class="filter">
+      <%= form_for @filter, as: '', url: school_participants_path(@school), method: :get do |f| %>
+        <%= f.govuk_error_summary %>
```

Comment:
I don't think we need this error summary - as it should be impossible to get the form in an error state (as the first radio option is always selected by default)?

```suggestion
```